commit 51d71ed3afd185d0a0fd2172f0becef2b26a5282
Author: Palmer Dabbelt <palmer@dabbelt.com>
Date:   Fri Mar 28 10:13:16 2014 -0700

    Fix p{perl,bash}c shebang on prefix
    
    When I merged the script compilers I forgot to make the Gentoo prefix
    ebuild work.

diff --git a/src/pbashc.c b/src/pbashc.c
index e8eeb75..045ad9b 100644
--- a/src/pbashc.c
+++ b/src/pbashc.c
@@ -25,15 +25,17 @@
 #include <stdlib.h>
 #include <string.h>
 
-#ifndef SHEBANG
+#ifndef SHEBANG_PREFIX
+#define SHEBANG_PREFIX ""
+#endif
+
 #if defined(LANG_PERL)
-#define SHEBANG "#!/usr/bin/perl\n"
+#define SHEBANG "#!" SHEBANG_PREFIX "/usr/bin/perl\n"
 #elif defined(LANG_BASH)
-#define SHEBANG "#!/bin/bash\n"
+#define SHEBANG "#!" SHEBANG_PREFIX "/bin/bash\n"
 #else
 #error "No language defined"
 #endif
-#endif
 
 /* The output file is a global file. */
 static FILE *outfile;
